src/core/openthread-core-default-config.h

```diff
@@ -159,6 +159,16 @@
 #define OPENTHREAD_CONFIG_MAX_TX_ATTEMPTS_INDIRECT_POLLS 4
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_TX_NUM_BCAST
+ *
+ * The transmit number of a broadcast frame in MAC layer.
```

Comment:
suggestion:
```
The number of times each IEEE 802.15.4 broadcast frame is transmitted.

The minimum value is 1.  Values larger than 1 may improve broadcast reliability by increasing redundancy, but may also increase congestion.
```

Reply:
Updated. Please review it at your convenience.